examples/aws/verify_egressv2.go

```diff
@@ -3,33 +3,24 @@ package aws
 import (
 	"context"
 	"fmt"
-	"os"
 	"time"
 
-	"github.com/aws/aws-sdk-go-v2/credentials"
 	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
 	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
 )
 
 func extendValidateEgressV2() {
```

Comment:
so, I can see we no longer need v1/v2 differentation? https://github.com/openshift/osd-network-verifier/blob/main/docs/aws/aws.md#112-go-implementation-examples

Is this change suitable w/ ocm-backend? I think it is worth exploring/ pinging them to ask we're changing the way of client generation.
 https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/cmd/clusters-service/service/aws/inflight_service.go#L9
&
 https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/cmd/clusters-service/service/aws/inflight_service.go#L229

Reply:
`Is this change suitable w/ ocm-backend?` :  Another great point. I updated the function signature to allow modifying args list without breaking dependencies in future. I'll contact ocm backend team to update their code once we merge this, and before moving to prod. No point contacting people until our change is finalized.